examples/src/main/python/hbase_inputformat.py

```diff
@@ -72,6 +75,9 @@
         keyConverter=keyConv,
         valueConverter=valueConv,
         conf=conf)
+    # hbase_rdd is a RDD[dict]
```

Comment:
I think they should have the same meaning, `RDD[dict]` is only a notation in Scala, `RDD of dict` is plain English.